README.md

```diff
@@ -1054,7 +1054,7 @@ Games, game servers and control panels.
 
 - [Mindmaps](https://github.com/drichard/mindmaps) - Open source, offline capable, mind mapping application. ([Demo](https://www.mindmaps.app)) `AGPL-3.0` `HTML5`
 - [My Mind](https://github.com/ondras/my-mind) - Web application for creating and managing mind maps. ([Demo](https://my-mind.github.io/?url=examples%2Ffeatures.mymind)) `MIT` `Javascript`
-- [Weaviate](https://github.com/semi-technologies/weaviate) - A cloud-native, realtime vector search engine integrating scalable machine learning models (GraphQL and RESTful APIs). ([Demo](https://www.semi.technology/documentation/weaviate/current/)) `BSD-3-Clause` `Go`
+- [Weaviate](https://github.com/semi-technologies/weaviate) - A cloud-native, realtime vector search engine integrating scalable machine learning models (GraphQL and RESTful APIs). ([Docs](https://weaviate.io/developers/weaviate/current/), [Demos](https://github.com/semi-technologies/awesome-weaviate)) `BSD-3-Clause` `Go`
```

Comment:
Format your submission as follows, where Demo and Clients are optional.
Do not add a duplicate Source code link if it is the same as the main link.
Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
for it, even if the project's webpage or readme uses another capitalisation
such as title case, all caps, small caps or all lowercase.
Demo links should only be used for interactive demos, i.e. not video demonstrations.
```
- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language`
```

- Please remove the Docs link
- `Demos` should be singular `Demo`

Thanks

Reply:
done